Tests/Reqnroll.VisualStudio.Specs/Features/Editor/Commands/FindUnusedStepDefinitionCommand.feature

```diff
@@ -0,0 +1,90 @@
+Feature: Find unused step definitions command
+
+Rules:
```

Comment:
Now Gherkin has a `Rule` keyword. Unfortunately the find usages feature file was not refactored yet, but you can look at for example the [GoToHooksCommand.feature](https://github.com/reqnroll/Reqnroll.VisualStudio/blob/main/Tests/Reqnroll.VisualStudio.Specs/Features/Editor/Commands/GoToHooksCommand.feature) file.

In your case I would do it this way:

```
Feature: Find unused step definitions command

List unused step definitions and allows jumping to the step definition method

Rule: Finds step definitions that are not matching to any feature steps

<first scenario>
<third scenario>

Rule: Finds unused step definitions with multiple binding attributes, listing only those that are unused

<second scenario>
```

Reply:
Done